embassy-boot/nrf/memory.x

```diff
@@ -0,0 +1,18 @@
+MEMORY
```

Comment:
What is the difference between this and `memory_bm.x`?

Reply:
None really. Maybe it's better to remove this and have a user needing to copy memory-bm.x to memory.x before compiling? Having this present just makes it easier to get started with the examples I guess.